src/components/lib/posix/posix.c

```diff
@@ -154,7 +148,7 @@ void *
 cos_mremap(void *old_address, size_t old_size, size_t new_size, int flags)
 {
 	printc("mremap not implemented!\n");
-	return -ENOTSUP;
+	return MAP_FAILED;
```

Comment:
Makes sense. 
My bad, I didn't look at the expanded view, it looked to me like it returned `int`. 
Is it possible to set errno here to indicate `-ENOTSUP`?